marketplace/collect.schema.json

```diff
@@ -30,14 +28,16 @@
                     "type": "boolean"
                 },
                 "language": {
-                    "type": "string"
+                    "type": "string",
+                    "enum": ["fr", "es", "en", "nl", "de"]
                 },
                 "format": {
                     "type": "object",
                     "additionalProperties": false,
                     "properties": {
                         "type": {
-                            "type": "string"
+                            "type": "string",
```

Comment:
Actually, you can remove `"type": "string"`. The `"enum": ["csv"]` is enough. The value can only be the string `"csv"` and nothing else.